sirepo/pkcli/job_agent.py

```diff
@@ -137,6 +137,11 @@ def __init__(self):
             fastcgi_error_count=0,
         )
 
+    def fastcgi_destroy(self):
+        self._fastcgi_file and pkio.unchecked_remove(self._fastcgi_file)
```

Comment:
This is an interesting question. I'll let it go. I personally don't mind the style, but would be curious what @moellep and @mkeilman (heebee geebees?)

Reply:
I've [used it elsewhere](https://github.com/radiasoft/sirepo/blob/eeba8cfb8743fefaf31034b24f6ff80ad4e2035e/sirepo/job_supervisor.py#L923). I don't have a strong feeling towards it. I like fewer if's but maybe this ins't worth it. In the linked example I think it could be confusing because one might read it and think self.op_slot is a bool when in fact it is the slot_proxy object. This one we don't care about the "return" value so maybe it matters less here.

Reply:
Personally, I prefer the "if" and in python it is just an extra line:
```python
if self._fastcgi_file:
    pkio.unchecked_remove(self._fastcgi_file)
```
In perl it was annoying to write:
```perl
if ($my_condition) {
    do_something();
}
```
and instead you could write:
```perl
do_something()
    if $my_condition;
```
which I didn't like because it is backwards, but at least it didn't require the curly brackets.

Reply:
That makes sense. I'm in agreement. Unless @mkeilman or @robnagler have strong thoughts if's from now on.

Reply:
Guess I'm an "if" guy